aws_doc_sdk_examples_tools/doc_gen.py

```diff
@@ -179,6 +179,7 @@ def clone(self) -> "DocGen":
             snippets={},
             snippet_files=set(),
             cross_blocks=set(),
+            entities=self.entities,
```

Comment:
(nit) I'd put this at LL178, and then move errors down, so that it's "The things we're keeping" and then "then things we're resetting"

Reply:
Updated.